### PR TITLE
Set max-width on ProductGrid

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Set max-width on ProductGrid

## Justify why they are needed
Make sure ProductGrid aligns to the rest of the page
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1910** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1910" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1910?utm_source=stack-comment).

File: apps/store/src/components/ProductGrid/ProductGrid.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import styled from '@emotion/styled'
 import { Fragment } from 'react'
 import { mq, Space, theme } from 'ui'
+import { MAX_WIDTH } from '../GridLayout/GridLayout'
```
Comment (robinandeer) [inline apps/store/src/components/ProductGrid/ProductGrid.tsx:4]: nit: update to `@/components` import

**Review by robinandeer — APPROVED** [review]
hmm just curious why we are not just using the grid which automatically applies the max width 🤔 

trying to understand if GridLayout could support more use cases!

**Comment by gustaveen** [comment]
> hmm just curious why we are not just using the grid which automatically applies the max width 🤔
> 
> trying to understand if GridLayout could support more use cases!

I hear you! I did this a s a first step since I haven't checked with Zak what logic we want to have for aligning different number of items. As of now the grid is setup with `auto-fit`, but I'll look into how we can align it to the 12-column grid as a next step 👍 @robinandeer

**Pull request merged**